spec/fixtures/weam.csv

```diff
@@ -1,3 +1,3 @@
 146,ASSOCIATION OF COMMUNICATIONS & ELECTRONICS,5241 Princess Anne Road,Suite 110,,VIRGINIA BEACH,VA,23462,USA,1587,,N,,,No,No,No,No,No,,,,5241 Princess Anne Road,Suite 110,,VIRGINIA BEACH,VA,23462,USA,1518,,,,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             ,
-10000008,MILTON POLICE DEPARTMENT,101 Federal Street,,,MILTON,DE,19968,USA,1740,N,N,APRVD,Educational Institution is Approved for All Chapters Except Chapter 31,No,Yes,No,No,No,,,N,101 Federal Street,,,MILTON,DE,19968,USA,1671,N,N,N,N                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          ,
+10000008,MILTON POLICE DEPARTMENT,101 Federal Street,,,MILTON,DE,19968,USA,1740,N,N,APRVD,Educational Institution is Approved for All Chapters Except Chapter 31,No,Yes,No,No,No,,,N,101 Federal Street,,,MILTON,DE,19968,USA,1671,N,N,N,N,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          ,

```

Comment:
Is the typical value for `Preferred Provider` in a WEAM upload going to be blank, like you left it here?
Or should it be `N` or `No`?

Reply:
It is going to be N, let me update that.